src/assets/json/data.json

```diff
@@ -1,1107 +1,1110 @@
+              "type": "radio"
             },
             {
-              "value":"c",
-              "lable": "contains url",
-              "type":"radio"
+              "id": 3,
+              "label": "contains url",
+              "type": "radio"
             },
             {
-              "value":"d",
-              "lable": "contains contact details for author",
-              "type":"radio"
+              "id": 4,
+              "label": "contains contact details for author",
+              "type": "radio"
             }
           ],
-          "answer": ["d"]
+          "answer": { "id": 4 }
         },
         {
-          "id": "Q-9.",
+          "questionId": "Q-9.",
           "question": " Which tag is used to create a dropdown in HTML Form?",
           "options": [
             {
-              "value":"a",
-              "lable": "<input>",
-              "type":"radio"
+              "id": 1,
+              "label": "<input>",
+              "type": "radio"
             },
             {
-              "value":"b",
-              "lable": "<select>",
-              "type":"radio"
+              "id": 2,
+              "label": "<select>",
+              "type": "radio"
             },
             {
-              "value":"c",
-              "lable": "<text>",
-              "type":"radio"
+              "id": 3,
+              "label": "<text>",
+              "type": "radio"
             },
             {
-              "value":"d",
-              "lable": "<textarea>",
-              "type":"radio"
+              "id": 4,
+              "label": "<textarea>",
+              "type": "radio"
             }
           ],
-          "answer": ["b"]
+          "answer": { "id": 2 }
         },
         {
-          "id": "Q-10.",
+          "questionId": "Q-10.",
           "question": "Which of the following CSS style property is used to specify an italic text?",
           "options": [
             {
-              "value":"a",
-              "lable": "style",
-              "type":"radio"
+              "id": 1,
+              "label": "style",
+              "type": "radio"
             },
             {
-              "value":"b",
-              "lable": "font",
-              "type":"radio"
+              "id": 2,
+              "label": "font",
+              "type": "radio"
             },
             {
-              "value":"c",
-              "lable": "font-style",
-              "type":"radio"
+              "id": 3,
+              "label": "font-style",
+              "type": "radio"
             },
             {
-              "value":"d",
-              "lable": "@font-face",
-              "type":"radio"
+              "id": 4,
+              "label": "@font-face",
+              "type": "radio"
             }
           ],
-          "answer": ["c"]
+          "answer": { "id": 3 }
         },
         {
-          "id": "Q-11.",
+          "questionId": "Q-11.",
```

Comment:
what is use of this question id ?

Reply:
There was no use of questionId hence removed